ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/BaseCommand.java

```diff
@@ -45,4 +48,14 @@
   )
   protected boolean dryRun = false;
 
+  protected static final Logger LOGGER = LoggerFactory.getLogger(Migrations.class);
+
+  @Override
+  public void run() {
+    final long startTime = System.nanoTime();
+    command();
+    LOGGER.info("Execution time: " + (System.nanoTime() - startTime) / 1000000000);
+  }
+
+  protected abstract void command();
```

Comment:
Good call making this protected. Let's update the implementations in the subclasses to not be public?